win32ss/user/ntuser/class.c

```diff
@@ -1275,39 +1293,47 @@ IntGetAtomFromStringOrAtom(
     if (ClassName->Length != 0)
     {
         WCHAR szBuf[65];
-        PWSTR AtomName;
+        PWSTR AtomName = szBuf;
         NTSTATUS Status;
 
         *Atom = 0;
 
         /* NOTE: Caller has to protect the call with SEH! */
-
-        if (ClassName->Length != 0)
+        _SEH2_TRY
```

Comment:
While this implementation should not be prone to the issue I mentioned above, I still don't like wrapping everything in SEH, just to safe a few lines of code when it comes to cleanup-

Reply:
I reduced the SEH scope